synadm/cli/regtok.py

```diff
@@ -0,0 +1,160 @@
+    if regtoks is None:
+        click.echo("Registration tokens could not be fetched.")
+        raise SystemExit(1)
+    if "registration_tokens" not in regtoks:
+        # Display error
+        helper.output(regtoks)
+    elif helper.output_format == "human":
+        if regtoks["registration_tokens"] == []:
+            click.echo("No registration tokens.")
+        else:
+            helper.output(regtoks["registration_tokens"])
+    else:
+        helper.output(regtoks)
+
+
+@regtok.command(name="details")
+@click.argument("token", type=str)
+@click.option(
+    "--datetime/--timestamp", "-d/-t", default=True, show_default=False,
+    help="""Display expiry time in a human readable format, or as a unix
+    timestamp in milliseconds.  [default: datetime].""")
+@click.pass_obj
+def regtok_details_cmd(helper, token, datetime):
+    """ View details of the given token
+    """
+    regtok = helper.api.regtok_details(token, datetime)
+    if regtok is None:
+        click.echo("Registration token could not be fetched.")
+        raise SystemExit(1)
+    helper.output(regtok)
+
+
+@regtok.command(name="new")
+@click.option(
+    "--token", "-t", type=str, default=None,
+    help="""Set the registration token. The default is a random value
+    generated by the server.""", show_default=True)
+@click.option(
+    "--length", "-l", type=int, default=16,
+    help="""The length of the randomly generated token if the token is not
+    specified.""", show_default=True)
+@click.option(
+    "--uses-allowed", "-u", type=int, default=None,
+    help="""The number of times the token can be used to complete a
+    registration before it becomes invalid.  [default: unlimited uses]""",
+    show_default=True)
+@click.option(
+    "--expiry-ts", "-t", type=int, default=None,
+    help="""The latest time the registration token is valid.
+    Given as the number of milliseconds since 1970-01-01 00:00:00 UTC.
+     [default: no expiry]""",
+    show_default=True)
+@click.option(
+    "--expire-at", "-e", type=click.DateTime(), default=None,
+    help="""The latest time the registration token is valid.
+    See above for available date/time formats.  [default: no expiry]""",
+    show_default=True)
+@click.pass_obj
+def regtok_new(helper, token, length, uses_allowed, expiry_ts, expire_at):
+    """ Create a new registration token
+    """
+    regtok = helper.api.regtok_new(
+        token, length, uses_allowed, expiry_ts, expire_at
+    )
+    if regtok is None:
+        click.echo("Registration token could not be created.")
+        raise SystemExit(1)
+    helper.output(regtok)
+
+
+@regtok.command(name="update")
+@click.argument("token", type=str)
+@click.option(
+    "--uses-allowed", "-u", type=int, default=None,
+    help="""The number of times the token can be used to complete a
+    registration before it becomes invalid. Use -1 for an unlimited
+    number of uses.  [default: unchanged]""",
+    show_default=True)
```

Comment:
or maybe you just get rid of the show_default flag everywhere you already hardcoded it? not sure but actually the default is False anyway.